crates/apollo-parser/test_data/parser/err/0034_directive_with_unfinished_string_value.txt

```diff
@@ -0,0 +1,116 @@
+                                    - ENUM_VALUE@135..137
+                                        - NAME@135..137
+                                            - IDENT@135..137 "ID"
+                            - COMMA@137..138 ","
+                            - WHITESPACE@138..141 "\n  "
+                            - ENUM_VALUE@141..145
+                                - NAME@141..145
+                                    - IDENT@141..145 "type"
+                            - WHITESPACE@145..146 " "
+                            - ENUM_VALUE@146..156
+                                - NAME@146..156
+                                    - IDENT@146..152 "String"
+                                    - COMMA@152..153 ","
+                                    - WHITESPACE@153..156 "\n  "
+                            - ENUM_VALUE@156..165
+                                - NAME@156..165
+                                    - IDENT@156..165 "modelCode"
+                            - WHITESPACE@165..166 " "
+                            - ENUM_VALUE@166..176
+                                - NAME@166..176
+                                    - IDENT@166..172 "String"
+                                    - COMMA@172..173 ","
+                                    - WHITESPACE@173..176 "\n  "
+                            - ENUM_VALUE@176..185
+                                - NAME@176..185
+                                    - IDENT@176..185 "brandName"
+                            - WHITESPACE@185..186 " "
+                            - ENUM_VALUE@186..196
+                                - NAME@186..196
+                                    - IDENT@186..192 "String"
+                                    - COMMA@192..193 ","
+                                    - WHITESPACE@193..196 "\n  "
+                            - ENUM_VALUE@196..206
+                                - NAME@196..206
+                                    - IDENT@196..206 "launchDate"
+                            - WHITESPACE@206..207 " "
+                            - ENUM_VALUE@207..214
+                                - NAME@207..214
+                                    - IDENT@207..213 "String"
+                                    - WHITESPACE@213..214 "\n"
+                            - WHITESPACE@214..215 "\n"
+- ERROR@96:108 "expected a closing \" for the string value" "@external])
+- ERROR@124:125 "expected a valid Value got At" @
+- ERROR@128:129 "expected a valid Value got LParen" (
+- ERROR@135:136 "expected a valid Value got Colon" :
+- ERROR@141:142 "expected a valid Value got RParen" )
+- ERROR@153:154 "expected }" !
+- ERROR@153:154 "expected a valid Value got Bang" !
+- ERROR@162:163 "expected a valid Value got Colon" :
+- ERROR@183:184 "expected a valid Value got Colon" :
+- ERROR@204:205 "expected a valid Value got Colon" :
+- ERROR@226:227 "expected a valid Value got Colon" :
+- ERROR@235:236 "expected a valid Value got RCurly" }
```

Comment:
something is off here, we should not be having this many "expected a valid Value" here. this should only happen for this bit of your graphql `import: ["@key", "@external]`.